reference-architecture/aws-ansible/README.md

```diff
@@ -65,8 +65,25 @@ scripts:
 export AWS_ACCESS_KEY_ID=foo
 export AWS_SECRET_ACCESS_KEY=bar
 ```
+
+### GitHub Authentication
+If using GitHub authentication (default) the file openshift-setup.yaml must be modified.
+Provide the OAuth clientID and clientSecret provided by GitHub. The clientID and clientSecret *MUST* be set or authentication will not be possible. Also,
+it is encouraged to provide an organization if not all of GitHub users will be able to login to the environment. For more information on GitHub authentication
+red through the reference architecture pdf.
+```
+vi openshift-ansible-contrib/reference-architecture/aws-ansible/playbooks/openshift-setup.yaml
```

Comment:
We should avoid having users customize playbooks that are shipped as part of this, rather these should be exposed as required variables passed in.